scrapyrt/core.py

```diff
@@ -272,10 +273,11 @@ def create_spider_request(self, kwargs):
             # It is needed because in POST handler we can pass
             # all possible requests kwargs, so it is easy to make mistakes.
             message = "Error while creating Request, {}".format(e.message)
+            log.msg(message, level=logging.WARNING)
             raise Error('400', message=message)
 
         req.dont_filter = True
         msg = u"Created request for spider {} with url {} and kwargs {}"
         msg = msg.format(self.spider_name, url, repr(kwargs))
-        log.msg(msg)
+        log.msg(msg, level=logging.DEBUG)
```

Comment:
I'd prefer to keep `log.msg` and `log.err` for backwards compatibility and switch to `logging`. Keep in mind that ScrapyRT was developed before Scrapy 1.0 release with logging support, so logging handling in ScrapyRT is a bit outdated and can be improved.